doc/driver.mld

```diff
@@ -120,7 +120,7 @@ Please note: it's only necessary to link the non-hidden modules (i.e., without a
 {ol {- Generation: odoc html-generate
 }}
 
-Once the compile and link phases are complete, the resulting [odocl] files may be rendered in a variety of formats. In this example we output HTML.
+Once the compile and link phases are complete, the resulting [odocl] files may be rendered in a variety of formats. In this example we output HTML:
```

Comment:
I find the `:` confusing. What about

```suggestion
Once the compile and link phases are complete, the resulting [odocl] files may be rendered in a variety of formats. In the driver exemplified in this document, we output HTML.
```

Reply:
I guess my comment never sent yesterday!

That was my mistake with the`:` -- I guess I saw a phantom example after it! 🙂 

What about: "For the driver exemplified herein, we output HTML."